Example/ViewController.swift

```diff
@@ -473,7 +483,10 @@ class ViewController: UIViewController {
 
     func navigationService(route: Route, routeIndex: Int, options: RouteOptions) -> NavigationService {
         let mode: SimulationMode = simulationButton.isSelected ? .always : .onPoorGPS
-        
+        if simulationButton.isSelected {
+            navigationMapView.mapView.locationManager.locationProvider.stopUpdatingHeading()
+            navigationMapView.mapView.locationManager.locationProvider.stopUpdatingLocation()
+        }
```

Comment:
It would be surprising if merely getting a navigation service – as the method name implies – has a side effect of pausing location updates going to the map view.

Reply:
I'll remove this part from this scope. Thanks for the suggestion.